[2023-04-26T18:56:22.658+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag_api.create_table_postgres manual__2023-04-26T18:56:16.668073+00:00 [queued]>
[2023-04-26T18:56:22.732+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag_api.create_table_postgres manual__2023-04-26T18:56:16.668073+00:00 [queued]>
[2023-04-26T18:56:22.735+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-26T18:56:22.740+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-04-26T18:56:22.741+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-26T18:56:22.800+0000] {taskinstance.py:1309} INFO - Executing <Task(PostgresOperator): create_table_postgres> on 2023-04-26 18:56:16.668073+00:00
[2023-04-26T18:56:22.814+0000] {standard_task_runner.py:55} INFO - Started process 2414 to run task
[2023-04-26T18:56:22.831+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'weather_dag_api', 'create_table_postgres', 'manual__2023-04-26T18:56:16.668073+00:00', '--job-id', '461', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag_api.py', '--cfg-path', '/tmp/tmpwk0r1tpj']
[2023-04-26T18:56:22.834+0000] {standard_task_runner.py:83} INFO - Job 461: Subtask create_table_postgres
[2023-04-26T18:56:23.165+0000] {task_command.py:389} INFO - Running <TaskInstance: weather_dag_api.create_table_postgres manual__2023-04-26T18:56:16.668073+00:00 [running]> on host b46a99c748bd
[2023-04-26T18:56:23.812+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=weather_dag_api
AIRFLOW_CTX_TASK_ID=create_table_postgres
AIRFLOW_CTX_EXECUTION_DATE=2023-04-26T18:56:16.668073+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-26T18:56:16.668073+00:00
[2023-04-26T18:56:23.818+0000] {sql.py:254} INFO - Executing:  CREATE TABLE IF NOT EXISTS measurements (
			timestamp TIMESTAMP,
			temp FLOAT,
			precipitation FLOAT,
			wind_speed FLOAT,
			location VARCHAR);
[2023-04-26T18:56:23.945+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-04-26T18:56:26.232+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-04-26T18:56:26.258+0000] {sql.py:375} INFO - Running statement:  CREATE TABLE IF NOT EXISTS measurements (
			timestamp TIMESTAMP,
			temp FLOAT,
			precipitation FLOAT,
			wind_speed FLOAT,
			location VARCHAR);, parameters: None
[2023-04-26T18:56:26.299+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=weather_dag_api, task_id=create_table_postgres, execution_date=20230426T185616, start_date=20230426T185622, end_date=20230426T185626
[2023-04-26T18:56:26.620+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-26T18:56:26.971+0000] {taskinstance.py:2596} INFO - 5 downstream tasks scheduled from follow-on schedule check
